Capstrone/assessmentportal/src/Quiz/AddQuiz.jsx

```diff
@@ -219,10 +219,19 @@ const cancelAddQuiz = () => {
                                 placeholder="Enter topic description"
                                 onChange={changeData}
                             ></textarea>
-                           
-                            <br /><br />
+                            <div>
+                                <label className="head3">Time</label><br /><br />
+                                <input
```

Comment:
A common component for input, table, form, cards, buttons, etc. can be used more.

Reply:
Created components for input,button and testarea